pkg/minikube/registry/drvs/podman/podman.go

```diff
@@ -104,7 +107,9 @@ func status() registry.State {
 		}
 
 		if v.LT(minReqPodmanVer) {
-			glog.Warningf("Warning ! minimum required version for podman is %s. your version is %q. minikube might not work. use at your own risk. To install latest version please see https://podman.io/getting-started/installation.html ", minReqPodmanVer.String(), v.String())
+			glog.Warningf("Warning ! minimum required version for podman is %s. your version is %q. minikube might not work. use at your own risk. To install latest version please see https://podman.io/getting-started/installation.html", minReqPodmanVer.String(), v.String())
```

Comment:
glog.Warrningf wont show to the user, please use the UI

out.WarningT("")

Reply:
Ah, sorry, I saw the request before but I didn't understand why.